Resources/config/schema/routing-1.0.xsd

```diff
@@ -83,7 +83,6 @@
         <xsd:attribute name="enabled" type="xsd:boolean" />
         <xsd:attribute name="manager-name" type="xsd:string" />
         <xsd:attribute name="content-basepath" type="xsd:string" />
-        <xsd:attribute name="use-sonata-admin" type="enable_auto" />
```

Comment:
makes sense. except if we rename it now people still have to update their IDE. if you want to do it we can, but that should not block this PR then.